components/service/telemetry/src/main/java/org/mozilla/telemetry/ping/TelemetryPocketEventPingBuilder.java

```diff
@@ -0,0 +1,57 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.telemetry.ping;
+
+import org.mozilla.telemetry.config.TelemetryConfiguration;
+import org.mozilla.telemetry.measurement.CreatedTimestampMeasurement;
+import org.mozilla.telemetry.measurement.EventsMeasurement;
+import org.mozilla.telemetry.measurement.LocaleMeasurement;
+import org.mozilla.telemetry.measurement.OperatingSystemMeasurement;
+import org.mozilla.telemetry.measurement.OperatingSystemVersionMeasurement;
+import org.mozilla.telemetry.measurement.PocketIdMeasurement;
+import org.mozilla.telemetry.measurement.ProcessStartTimestampMeasurement;
+import org.mozilla.telemetry.measurement.SequenceMeasurement;
+import org.mozilla.telemetry.measurement.TimezoneOffsetMeasurement;
+
+/**
+ * A telemetry ping builder for events of type "fire-tv-events".
+ *
+ * See the schema for more details:
+ *   https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/dc458113a7a523e60a9ba50e1174a3b1e0cfdc24/schemas/pocket/fire-tv-events/fire-tv-events.1.schema.json
+ */
+public class TelemetryPocketEventPingBuilder extends TelemetryPingBuilder {
```

Comment:
Can we add a unit test for this class? Take a look at the other ping builder tests. Just some sanity checks would be good. And for this ping builder in particular we may want to explicitly test that it does **not** contain the client id.